sdks/go/pkg/beam/core/runtime/exec/plan.go

```diff
@@ -87,6 +87,11 @@ func (p *Plan) ID() string {
 	return p.id
 }
 
+// ID returns the plan identifier.
```

Comment:
If we keep this method as is, the documentation needs to be corrected.
```suggestion
// Source returns the DataSource for this plan.
```

Reply:
Fixed below.